ci/plans/ceramic.cue

```diff
@@ -67,7 +67,8 @@ dagger.#Plan & {
 	client: network: "unix:///var/run/docker.sock": connect: dagger.#Socket
 
 	actions: {
-		_repo:        "js-ceramic"
+		_repo_ceramic:        "js-ceramic"
```

Comment:
Mind removing the extra whitespace before `js-ceramic` so that it's aligned with `composedb`?

Reply:
Installed the formatter!